src/kfs_frontend/kfs_utils.cpp

```diff
@@ -189,8 +189,11 @@ Status getRawInputContentsBatchSizeAndWidth(const std::string& buffer, int32_t&
         offset += (sizeof(uint32_t) + inputSize);
         tmpBatchSize++;
     }
-    if (offset != buffer.size()) {
-        SPDLOG_DEBUG("Raw input contents invalid format. Every input need to be preceded by four bytes of its size.");
+    if (offset > buffer.size()) {
+        SPDLOG_DEBUG("Raw input contents invalid format. Every input need to be preceded by four bytes of its size. Buffer exceeded by {} bytes", offset - buffer.size());
```

Comment:
Is it the case also for BS1?

Reply:
If we want to follow triton's binary extension, we need to include 4 bytes length even for BS=1